src/components/inputs/TextField.js

```diff
@@ -369,7 +365,7 @@ export default class TextField extends BaseInput {
 
   renderTitle() {
     const {floatingPlaceholder, title, titleColor, titleStyle} = this.getThemeProps();
-    const color = this.getStateColor(titleColor);
+    const color = this.getStateColor(titleColor || {focus: Colors.blue30, default: Colors.grey30});
```

Comment:
Why not use DEFAULT_PLACEHOLDER_COLOR here as well?

Reply:
done